decimal-number.c

```diff
@@ -0,0 +1,92 @@
+/*
+Module: Binary-Zeros-Ones-of-Decimal-Number.c
+
+Function:
+    Finding Binary zeros and Ones in a given decimal  number.
+
+Copyright notice:
+    This file copyright (C) 2022 by
+    MCCI Corporation
+    3520 Krums Corners Road
+    Ithaca, NY 14850
+    An unpublished work. All rights reserved.
+    This file is proprietary information, and may not be disclosed or
+    copied without the prior permission of MCCI Corporation.
+
+Author:
+    Athisankar, MCCI Corporation   May 2022
+*/
+
+#include<stdio.h>
+
+/****************************************************************************\
+|
+|   Code.
+|
+\****************************************************************************/
+
+/*
+Name: main()
+
+Function:
+    To compute Binary zeros and Ones in a given decimal  number.
+ and to diplay the output.
+
+Definition:
+    Void Function();
```

Comment:
`void` is a datatype and it has its standard format, mentioning with upper case is not standard, also that does not match actual definition